MediaWikiKit/MediaWikiKit/MWKSection.m

```diff
@@ -106,6 +106,10 @@ - (MWKTitle*)sourceTitle {
     }
 }
 
+- (BOOL)hasTextData {
+    return [self.article.dataStore htmlFileExistsForSectionId:self.sectionId title:self.article.title];
+}
```

Comment:
minor question, is it possible to simplify to `hasHTMLFileForSection:(MWKSection*)` and query the section's `id` and `title` to get the path to the file w/in the data store?